app/routes/languages.rb

```diff
@@ -6,9 +6,11 @@ class Languages < Core
       TOPICS = [:about, :exercises, :installing, :tests, :learning, :resources, :help, :launch].freeze
 
       get '/languages' do
-        tracks = X::Track.all
-        active, inactive = tracks.partition(&:active?)
-        planned, inactive = inactive.partition(&:planned?)
+        tracks = Trackler.tracks
+        active, inactive = tracks.partition { |t| t.active? }
+        planned = []
```

Comment:
> I also feel like we should be adding this functionality to Trackler

LOL. I added this to Trackler on the train this morning... before seeing this conversation. Let me ship that.

- Active: config says active, we trust that.
- Upcoming: someone actually did some work to add at least one exercise
- Planned: someone requested the language, I added a repo, but no exercises have been implemented

I chose "planned" rather than "requested" because it seems like it's more inviting for people to actually contribute.